test/conformance/api/v1/util.go

```diff
@@ -340,3 +344,22 @@ func validateImageDigest(imageName string, imageDigest string) (bool, error) {
 
 	return ref.Context().String() == digest.Context().String(), nil
 }
+
+// WaitForScaleToZero will wait for the specified deployment to scale to 0 replicas.
+// Will wait up to 6 times the ScaleToZeroGracePeriod (1 minute) before failing.
+func WaitForScaleToZero(t *testing.T, deploymentName string, clients *test.Clients) error {
+	t.Helper()
+	t.Logf("Waiting for %q to scale to zero", deploymentName)
+	ScaleToZeroGracePeriod := time.Minute
```

Comment:
is there another place we can get this ScaleToZeroGracePeriod value (aside from a configmap) or is this sufficient for a test?

Reply:
Probably a flag to the test, probably. Thus the executor of the test can change the value accordingly.

Reply:
It looks like tests flags are generally added in `e2e_flags.go`? 
So far, this has these items:
```
        ResolvableDomain bool   // Resolve Route controller's `domainSuffix`
	Https            bool   // Indicates where the test service will be created with https
	IngressClass     string // Indicates the class of Ingress provider to test.
	CertificateClass string // Indicates the class of Certificate provider to test.
	SystemNamespace  string // Indicates the system namespace, in which Knative Serving is installed.
```

Should we really add ScaleToZeroGracePeriod here?  I'm not sure there's a reason for this value to be overriden in the test case.

Reply:
Perhaps.
cc @chaodaiG @mattmoor